aries-site/src/data/wcag/components.json

```
@@ -862,5 +862,519 @@
+        "notes": ""
+      },
+      {
+        "rule": "3.1.2",
+        "label": "Language of Parts",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.1.3",
+        "label": "Unusual Words",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.1.4",
+        "label": "Abbreviations",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.1.5",
+        "label": "Reading Level",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.1.6",
+        "label": "Pronunciation",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.2.1",
+        "label": "On Focus",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "3.2.2",
+        "label": "On Input",
+        "status": "passed",
+        "notes": ""
+      },
+      {
+        "rule": "3.2.3",
+        "label": "Consistent Navigation",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.2.4",
+        "label": "Consistent Identification",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.2.5",
+        "label": "Change on Request",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.2.6",
+        "label": "Consistent Help",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.3.1",
+        "label": "Error Identification",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.3.2",
+        "label": "Labels or Instructions",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.3.3",
+        "label": "Error Suggestion",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.3.4",
+        "label": "Error Prevention (Legal, Financial, Data)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.3.5",
+        "label": "Help",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.3.6",
+        "label": "Error Prevention (All)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.3.7",
+        "label": "Redundant Entry",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.3.8",
+        "label": "Accessible Authentication (Minimum)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "3.3.9",
+        "label": "Accessible Authentication (Enhanced)",
+        "status": "not-applicable",
+        "notes": ""
+      },
+      {
+        "rule": "4.1.2",
+        "label": "Name, Role, Value",
+        "status": "conditional",
```

```suggestion
        "status": "failed",
```

I think this rule is failing see comment here: https://hpe.sharepoint.com/:x:/r/teams/hpe-design/Shared%20Documents/HPE%20Design%20System/Accessibility/WCAG%20Rule%20Evaluation%20Template.xlsx?d=we975e19f32644f4cbd0a446f23f89277&csf=1&web=1&e=qTSE4V&nav=MTBfezNhZWVjZTIwLTc0YzAtNDU5MS05ZWIzLTcwYTEyOTc1NDI2Zn1fezAzRTJCOTg2LTUzNzAtNEZCMy1CRkZGLTM5OUNCMjVBMTMwNX0